src/briefcase/commands/base.py

```diff
@@ -400,7 +399,6 @@ def bundle_path(self, app) -> Path:
         """
         return self.build_path(app) / self.output_format.lower()
 
-    @abstractmethod
     def binary_path(self, app) -> Path:
```

Comment:
I'm not sure why this was flagged as an issue to be cleaned up - it *is* an abstract method (there's currently documentation, but no implementation), and if the method isn't implemented, there's going to be some interesting fireworks from all the methods that use `binary_path`.